cryptography/src/secp256r1/mod.rs

```diff
@@ -4,7 +4,12 @@
 //! deterministic signatures as specified in [RFC 6979](https://datatracker.ietf.org/doc/html/rfc6979), and enforces
 //! signatures are normalized according to [BIP 62](https://github.com/bitcoin/bips/blob/master/bip-0062.mediawiki#low-s-values-in-signatures).
 //!
-//! # Example
+//! Two variations of this scheme are provided. The base scheme does not support public key recovery, while the
```

Comment:
how do you feel about calling this `standard` and` recoverable`?

Reply:
In other crates, we don't assume a "default" and explicitly class all variants (journal::variable and journal::fixed not journal and journal::fixed). Curious to get your thoughts on that here